addons/addon-base-raas/packages/base-raas-services/lib/schema/register-study.json

```diff
@@ -30,11 +32,13 @@
     "folder": {
       "type": "string",
       "minLength": 1,
-      "maxLength": 1000
+      "maxLength": 1000,
+      "pattern": "^([A-Za-z0-9-_ ]+)$"
```

Comment:
We might want to account for more characters and umlauts.  including '@'.  In SWB default buckets, some folders contain '@'.